### DISCUSSION TITLE
Scalar Api collection Genrator

### DISCUSSION THREAD

**Discussion by aditya05062002** [scalar/scalar#3860] [Q&A]
Can any one tell me how to create scalar api collection using C# and where i can find scalar API schema URL ? I am currently developing Scalar Collection and Document Genrator using C# could any one tell me where i refer ?

**Comment by hanspagel** [comment]
Sorry, I don’t have any experience with C#, but did you try to research how to create an OpenAPI/Swagger document? Scalar Collections are based on the OpenAPI standard, and can be created from and exported to OpenAPI documents.

**Comment by marclave** [comment]
@xC0dex might be able to help out here :)

**Comment by xC0dex** [comment]
Hey there!
I’d love to help you out, but I think I’m missing a piece of the puzzle here. When you say 'Scalar API Collection' are we talking about the generated OpenAPI document, or something else? 😄 Which document generator for .NET are you using? Any extra clues you can give would be awesome!

**Reply by aditya05062002** [reply]
It's like custom test data by that data it should create OpenAPI document according to that and using that OpenAPI document Scalar document or collection on Scalar client. 

For e.g : we have some data in json format according to that data postman collection is generated automatically for that test data like scripts and all etc.. 

just like that i want to create OpenAPI Document frist because Scalar use OpenAPI Doc and through that document Scalar Collection in Client and Scalar Documentation
